aspnetcore/migration/20_21.md

```diff
@@ -52,14 +51,19 @@ ASP.NET Core includes the following shared runtimes:
 * [Microsoft.AspNetCore.App](xref:fundamentals/metapackage-app)
 * [Microsoft.AspNetCore.All](xref:fundamentals/metapackage)
 
-Rules for projects targeting the shared runtime:
+The version specified by the package reference is the *minimum required* version. If a project references the 2.1.1 version of these packages, the app produced won't run on a machine that only has the 2.1.0 runtime installed.
+
+Known issues for projects targeting the shared runtime:
+
+* It's difficult to use the 2.1.0 `Microsoft.AspNetCore.App` package with Entity Framework Core 2.1.1. The recommended solution is to upgrade `Microsoft.AspNetCore.App` to 2.1.1. For more information, see [Packages that share dependencies with Microsoft.AspNetCore.App cannot reference patch versions](https://github.com/aspnet/Universe/issues/1180).
```

Comment:
What about "Using `Microsoft.AspNetCore.App` 2.1.0 with Entity Framework Core 2.1.1 causes NuGet restore failures due to package conflicts."?

Reply:
I like IT! :smile: